release-blocking-jobs.md

```diff
@@ -154,3 +155,23 @@ The general process to promote or demote jobs follows these steps:
 
 1. Open a kubernetes/test-infra PR that promotes or demotes the job to/from any
    of the sig-release boards.
+
+## Pull Request Blocking Jobs
```

Comment:
I remember a doc from Aaron with the criteria, so we only have only a subset of selected job, that run in less than one hour, high stability required, ...

Reply:
found it https://github.com/kubernetes/test-infra/issues/6380#issuecomment-887158532

Pasting for reference

```
Let's lift loosely from my https://github.com/kubernetes/sig-release/issues/24#issuecomment-344381137 and https://github.com/kubernetes/sig-release/pull/346 that turned into what [release-blocking criteria are today](https://github.com/kubernetes/sig-release/blob/master/release-blocking-jobs.md#release-blocking-criteria-and-dashboard)

Looking at the merge-blocking jobs we have today, I would hope they meet these criteria:

The job has a 90th percentile duration of 75min (ideally <60min)
The job is better than a coin flip over all PR's (<50% daily failure rate across all PRs)
The job is better than four coin flips against your PR (95% consistent aka 5% flake rate)
The job is supported by contributors who can bring it back to green within a business day if it starts failing continuously or very near so
If the job is e2e, it must use OSS to provision the cluster (no hosted solutions)
```

Reply:
can you incorporate that @BenTheElder ^^^ ?

Reply:
Ooh, thanks for digging that up!

I agree we should include more rigorous criteria and this seems like a good data-driven starting point.

Reply:
/hold
I'm going to come back to ping the thread again when I've incorporate this properly, for now going to get in #2229 first

Reply:
ping @BenTheElder can we put a closure on this?

Reply:
Please fork the PR if you have bandwidth, still too many things going on.